streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -192,11 +192,8 @@ class StreamingContext private[streaming] (
       None
     }
 
-  /** Register streaming source to metrics system */
+  /* Initializing a streamingSource to register metrics */
```

Comment:
It previously held the block of code that did the registration.
Here it simply initializes the streamingSource